easy-ui-react/src/styles/_media-queries.scss

```diff
@@ -0,0 +1,25 @@
+@use "@easypost/easy-ui-tokens/scss/variables" as *;
+
+@mixin breakpoint-sm-up {
+  @media (min-width: #{$ezui-breakpoint-sm}) {
+    @content;
+  }
+}
+
+@mixin breakpoint-md-up {
+  @media (min-width: #{$ezui-breakpoint-md}) {
+    @content;
+  }
+}
+
+@mixin ezui-tablet {
+  @include breakpoint-sm-up {
+    @content;
+  }
+}
+
+@mixin ezui-desktop {
+  @include breakpoint-md-up {
+    @content;
+  }
+}
```

Comment:
**thought:** while i like these shortcuts, i feel a little on the fence with them because `tablet` and `desktop` seem like arbitrary tags to me. it's not clear to me what defines `tablet` and `desktop` in the real world. unless we get strict labeling from Design, while it is slightly less friendly, i would personally prefer using the `breakpoint-sm-up` and `breakpoint-md-up` since i think it's more explicit in aligning with our breakpoints

but curious how you feel after reading that

Reply:
i implemented your suggestion, but i had defined the mixin based on a spec in figma regarding breakpoints:

![image](https://user-images.githubusercontent.com/44451175/235789899-1c3826fc-66ed-4497-a50d-983ae4efcece.png)

Reply:
oh i haven't seen this in Figma. when i last looked it was using the `xs`, `sm`, etc. interesting. "Web" doesn't quite make sense but i suppose desktop like you had it makes a reasonable alternative. it still feels slightly iffy to me but i'd be down to do it like this if you want. how do you think we would represent the different desktop sizes? t-shirt sizes? `desktop`, `desktop-lg`, `desktop-xl`, `desktop-2xl`, etc?

Reply:
hmm, i dont feel super strongly about including it and if youre iffy on it, id feel more comfortable if we leave it out of the mixin for now at least. probably something we can revisit when we're developing molecules or layout concerned components